CSharp/InvertedIndexSearcher/words/IWord.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace InvertedIndexSearcher.words
+{
+    public interface IWord
+    {
+        public string WordAsString { get; }
+    }
+}
```

Comment:
In Lazem Nabood Interface beshe chon word class data hast va functionality nadadre